packages/next-app/src/components/Table.tsx

```diff
@@ -103,7 +103,7 @@ function BodyCell({
     <td
       className={cn(
         "whitespace-nowrap text-sm text-gray-500",
-        customWidth ? cn(customWidth, "pl-4") : "p-4"
+        customWidth ? cn(customWidth, "pl-4") : "px-1 py-4"
```

Comment:
you can also just use custom width where you need -- remember this is going to affect all uses of the table component.